subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/initialization/ConfigurationCacheProblemsListener.kt

```diff
@@ -177,5 +177,5 @@ class DefaultConfigurationCacheProblemsListener internal constructor(
     fun isStableConfigurationCacheEnabled() = featureFlags.isEnabled(FeaturePreviews.Feature.STABLE_CONFIGURATION_CACHE)
 
     private
-    fun isExecutingTask() = taskExecutionTracker.currentTask.isPresent
+    fun isExecutingWork() = workExecutionTracker.currentTask.isPresent || workExecutionTracker.isExecutingTransformAction
```

Comment:
❓ Any reason not to reuse `hasRunningWork` here?

Reply:
Could you please elaborate a bit? I only see `OperationListener.hasRunningWork`, but it has different semantic - it checks if any thread is running some work, but we need to know only about the current thread. Or do you mean extract `isExecutingWork` into a common helper?

Reply:
Sorry, I misread the code.